Packs/Core/Playbooks/playbook-Suspicious_Hidden_User_Created.yml

```diff
@@ -0,0 +1,1263 @@
+              left:
+                value:
+                  simple: foundIncidents.severity
+                iscontext: true
+              right:
+                value:
+                  simple: LOW
+              ignorecase: true
+          - - operator: isNotEqualString
+              left:
+                value:
+                  simple: foundIncidents.CustomFields.action
+                iscontext: true
+              right:
+                value:
+                  simple: BLOCKED
+              ignorecase: true
+          accessor: cid
+          transformers:
+          - operator: uniq
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 230,
+          "y": 930
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "35":
+    id: "35"
+    taskid: e8cda6f4-6d4e-46fb-8ada-1b794d7caa27
+    type: title
+    task:
+      id: e8cda6f4-6d4e-46fb-8ada-1b794d7caa27
+      version: -1
+      name: Host Checks
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "3"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -170,
+          "y": 620
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "36":
+    id: "36"
+    taskid: 5d9c74e6-5058-410a-827a-324c9144abfe
+    type: regular
+    task:
+      id: 5d9c74e6-5058-410a-827a-324c9144abfe
+      version: -1
+      name: Terminate process manually
+      description: Please investigate the alerts related to this incident, and terminate the CGO (Causality Group Owner) process that caused the suspicious hidden user to be created.
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "25"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -180,
+          "y": 2390
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "12_22_Yes": 0.3,
+      "12_23_Yes": 0.48,
+      "12_24_#default#": 0.54,
+      "24_23_yes": 0.54,
+      "24_25_#default#": 0.16,
+      "30_25_#default#": 0.24,
+      "33_36_#error#": 0.61,
+      "6_7_yes": 0.51,
+      "6_9_#default#": 0.23
+    },
+    "paper": {
+      "dimensions": {
+        "height": 2595,
+        "width": 1660,
+        "x": -180,
+        "y": 60
+      }
+    }
+  }
+inputs: []
+outputs: []
+marketplaces: ["marketplacev2"]
+tests:
+- no tests
+fromversion: 6.10.0
```

Comment:
you use kill CGO, so you need to make from version 8.8.0 + 
look at other playbooks that kills CGO for reference